view/frontend/web/js/view/payment/method-renderer/two_payment.js

```diff
@@ -318,13 +318,65 @@ define([
                     quantity: item['qty'],
                     unit_price: parseFloat(item['price']).toFixed(2),
                     tax_amount: parseFloat(item['tax_amount']).toFixed(2),
-                    tax_rate: parseFloat(item['tax_percent']).toFixed(6),
+                    tax_rate: (parseFloat(item['tax_percent']) / 100 ).toFixed(6),
                     tax_class_name: '',
                     quantity_unit: config.orderIntentConfig.weightUnit,
                     image_url: item['thumbnail'],
                     type: item['is_virtual'] === '0' ? 'PHYSICAL' : 'DIGITAL'
                 });
             });
+
+            // create tax subtotals for order intent for additional logging
+            const calculateTaxSubtotals = (lineItems) => {
```

Comment:
should we define this function at upper level?

Reply:
moved to top